meson.build

```diff
@@ -263,7 +263,10 @@ if get_option('bash-completions')
 		'completions/bash/swaymsg',
 	)
 	if bash_comp.found()
-		bash_install_dir = bash_comp.get_pkgconfig_variable('completionsdir')
+		bash_install_dir = bash_comp.get_pkgconfig_variable(
+			'completionsdir',
+			define_variable: ['prefix', prefix]
```

Comment:
We'll want to define `datadir` here, not `prefix`. Here's the bash pc file:

```
datadir=/usr/share
completionsdir=${datadir}/bash-completion/completions
```

Reply:
Hm, my bash-completion.pc uses prefix: `completionsdir=${prefix}/share/bash-completion/completions`

I guess we can just send both variables...

Reply:
arg, I can't find a working syntax to send multiple variable?! what the heck, meson.

```
                bash_install_dir = bash_comp.get_pkgconfig_variable(
                        'completionsdir',
                        define_variable: [
                                'prefix', prefix,
                                'datadir', datadir,
                        ]
                )
->
../meson.build:267:2: ERROR: define_variable must be made up of 2 strings for VARIABLENAME and VARIABLEVALUE
```

Reply:
It seems like upstream uses `datadir`:

https://github.com/scop/bash-completion/blob/master/bash-completion.pc.in#L6

Reply:
Yeah I just checked that and was testing. pushed update into datadir